docs/src/pages/demos/dialogs/CustomizedDialog.tsx

```diff
@@ -21,95 +22,83 @@ const styles = (theme: Theme) =>
       top: theme.spacing(1),
       color: theme.palette.grey[500],
     },
-  });
+    content: {
+      padding: theme.spacing(2),
+    },
+    actions: {
+      margin: 0,
+      padding: theme.spacing(1),
+    },
+  }),
+);
 
-export interface DialogTitleProps extends WithStyles<typeof styles> {
-  id: string;
+interface DialogTitleProps {
   children: React.ReactNode;
-  onClose: () => void;
+  handleClose?(): void;
 }
 
-const DialogTitle = withStyles(styles)((props: DialogTitleProps) => {
-  const { children, classes, onClose } = props;
+function DialogTitle(props: DialogTitleProps) {
+  const classes = useStyles();
+
   return (
     <MuiDialogTitle disableTypography className={classes.root}>
-      <Typography variant="h6">{children}</Typography>
-      {onClose ? (
-        <IconButton aria-label="Close" className={classes.closeButton} onClick={onClose}>
+      <Typography variant="h6">{props.children}</Typography>
+      {props.handleClose && (
```

Comment:
Not sure why the lint rule didn't trigger but prefer destructuring. It minifies better (and is required for type narrowing).